sdk/nodejs/automation/localWorkspace.ts

```diff
@@ -356,7 +356,7 @@ export class LocalWorkspace implements Workspace {
 
             // Transform the serialized representation back to what we expect.
             for (const key of stackSettingsSerDeKeys) {
-                if (stackSettings.hasOwnProperty(key[0])) {
+                if (Object.hasOwn(stackSettings, key[0])) {
```

Comment:
This doesn't appear to be supported on Node 16.8: https://node.green/#ES2022-features-Object-hasOwn

I think we'd want to wait on this until sometime after we drop Node 16 support.

Reply:
No argument there!